test/ast.spec.js

```diff
@@ -466,6 +466,11 @@ describe('AST', () => {
             expect(getParsedSql(sql)).to.equal('SELECT NULL');
         });
 
+        it('should support trailing zeros', function() {
```

Comment:
Can you replace the function declaration by an arrow function?

Reply:
Done. However, [passing arrow functions to Mocha is discouraged](https://mochajs.org/#arrow-functions).